### PR TITLE
[#5] Connector: s3 file sink connector

### PR DESCRIPTION
**Pull request by PMjoydas**

### CODE REVIEW

File: components/Python/sink/s3_file_sink/component.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{
+  "engineType": "Python",
+  "userStandalone": false,
+  "language": "Python",
+  "name": "s3_file_sink",
+  "label": "S3 File Sink",
+  "description": "Save a file to an S3 bucket + key",
+  "program": "s3_file_sink.py",
+  "componentClass": "S3FileSink",
+  "group": "Connectors",
```
Comment (liororama) [inline components/Python/sink/s3_file_sink/component.json]: The group should be Sinks

File: components/Python/sink/s3_file_sink/component.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  "useMLOps": true,
+  "inputInfo": [{
+    "description": "File name",
```
Comment (liororama) [inline components/Python/sink/s3_file_sink/component.json:13]: File name to read content from

File: components/Python/sink/s3_file_sink/component.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    "label": "File-Name",
+    "defaultComponent": "",
```
Comment (liororama) [inline components/Python/sink/s3_file_sink/component.json:15]: Do we need the defaultComponent field?

File: components/Python/sink/s3_file_sink/s3_file_sink.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+        client = boto3.client(
+            's3',
+            aws_access_key_id=self._params["aws_access_key_id"],
+            aws_secret_access_key=self._params["aws_secret_access_key"],
+        )
+        data = open(file_path, 'rb')
+        client.put_object(Bucket=self._params["bucket"], Key=self._params["key"], Body=data)
+
+        return file_path
```
Comment (liororama) [inline components/Python/sink/s3_file_sink/s3_file_sink.py]: The return value of _save_file is not used

File: components/Python/sink/s3_file_sink/component.json (hunk 1 above)
Comment (PMjoydas) [reply components/Python/sink/s3_file_sink/component.json]: fixed changes to "Sinks"

File: components/Python/sink/s3_file_sink/component.json (hunk 2 above)
Comment (PMjoydas) [reply components/Python/sink/s3_file_sink/component.json:13]: changed to "File to read contents"

File: components/Python/sink/s3_file_sink/s3_file_sink.py (hunk 4 above)
Comment (PMjoydas) [reply components/Python/sink/s3_file_sink/s3_file_sink.py]: removed the return value of "file_path"

File: components/Python/sink/s3_file_sink/component.json (hunk 3 above)
Comment (PMjoydas) [reply components/Python/sink/s3_file_sink/component.json:15]: Not sure

**Review by sindhu-ghanta — APPROVED** [review]
I agree with Lior Amar's comments.

**Pull request merged**